client/plots/barchart.js

```diff
@@ -921,6 +933,16 @@ function setRenderers(self) {
 			.style('font-weight', 'normal')
 			.html(noSkipped ? '' : 'N/A: association test skipped because of limited sample size <br>')
 	}
+
+	// a function used by generatePvalueTable to negate a term label
+	// TODO: add more conditions for better negation than "not termLable"
+	function negateTermLabel(termLabel) {
+		if (termLabel.toUpperCase().startsWith('NOT ')) {
```

Comment:
can merge now but should continue to evaluate this logic. e.g. can the name `not something` ever be one of 3 groups?